plugins/backstage-plugin-flux/src/components/FluxRuntimeCard/FluxRuntimeCard.tsx

```diff
@@ -28,8 +32,32 @@ const FluxRuntimePanel: FC<{ many?: boolean }> = ({ many }) => {
  *
  * @public
  */
-export const FluxRuntimeCard = ({ many = true }: { many?: boolean }) => (
-  <WeaveGitOpsContext>
-    <FluxRuntimePanel many={many} />
-  </WeaveGitOpsContext>
-);
+export const FluxRuntimeCard = ({ many = true }: { many?: boolean }) => {
+  // Set both the garbage collection time and max-age to 1 hour
+  // gcTime should be higher than max-age to avoid removing things too soon.
+  const gcTime = 1000 * 60 * 60;
```

Comment:
Could we keep this for 24hr at least?

Flux isn't released very often, and the costs of being "out-of-date" aren't particularly high?